nBTC/sources/nbtc.move

```diff
@@ -217,6 +217,13 @@ fun init(witness: NBTC, ctx: &mut TxContext) {
     );
 }
 
+public fun new_utxo(tx_id: vector<u8>, vout: u32, value: u64): Utxo {
+    Utxo {
+        tx_id,
+        vout,
+        value,
+    }
```

Comment:
Maybe we should move it to btclib? See my earlier comment about btclib

Reply:
Yes, I created task for refactor